geniza/footnotes/models.py

```diff
@@ -0,0 +1,68 @@
+from django.db import models
+from django.contrib.contenttypes.models import ContentType
+from django.contrib.contenttypes.fields import GenericForeignKey
+
+from multiselectfield import MultiSelectField
+# https://pypi.org/project/django-multiselectfield/
+
+from geniza.people.models import Person
+
+class SourceType(models.Model):
+    type = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.type
+
+
+class Source(models.Model):
+    # TODO: Multiple authors
+    author = models.ForeignKey(Person, on_delete=models.CASCADE)
+    title = models.CharField(max_length=255)
+    year = models.PositiveIntegerField(blank=True, null=True)
+    # TODO: `edition_number` isn't a number. Can we call it `edition` ?
+    edition_number = models.CharField(max_length=255, blank=True)
+    # TODO: What kind of values would volume contain? Would they just be an int?
+    volume = models.CharField(max_length=255, blank=True)
+    page_range = models.CharField(max_length=255, blank=True,
+        help_text='The range of pages being cited. Do not include "p", "pg", etc. and follow the format # or #-#')
+    source_type = models.ForeignKey(SourceType, on_delete=models.CASCADE)
+    # TODO: (RR) Null? 
+    # TODO: (RR) Should there be a default language?
+    language = models.ForeignKey('corpus.LanguageScript', on_delete=models.SET_NULL,
+        help_text='In what language was the source published?', null=True)
+
+    class Meta:
+        ordering = ['author__sort_name']
+
+    def __str__(self):
+        # TODO: Generate citation
+        return self.title
```

Comment:
I think a short citation similar to the display in the metadata spreadsheet would be fine for now; no need to worry about generating a formatted citation